CHANGELOG.md

```diff
@@ -357,6 +357,15 @@ macOS < 12.
 
 #### Enhancements
 
+* Add new option `only_enforce_before_trivial_lines` to
+  `vertical_whitespace_closing_braces` rule. It restricts
+  the rule to apply only before trivial lines (containing
+  only closing braces, brackets and parentheses). This
+  allows empty lines before non-trivial lines of code
+  (e.g. if-else-statements).
```

Comment:
This line is now missing the two trailing spaces:
```suggestion
  (e.g. if-else-statements).  
```
Sorry for the nitpicking. 😬

Reply:
Done.